packaging/dbscripts/upgrade/04_05_0280_change_default_display.sql

```diff
@@ -0,0 +1,32 @@
+-- Removing all graphics from blank template
+DELETE FROM vm_device
+WHERE type = 'graphics'
+    AND vm_id = '00000000-0000-0000-0000-000000000000'
+    AND EXISTS (select 1 from vm_static where vm_guid = '00000000-0000-0000-0000-000000000000' AND default_display_type = 1);
+
+-- Add VNC graphics to blank template
+INSERT INTO vm_device (device_id, vm_id, type, device, address, spec_params, is_managed, is_plugged, is_readonly)
+SELECT uuid_generate_v1(), '00000000-0000-0000-0000-000000000000', 'graphics', 'vnc', '', '', true, true, false
+WHERE EXISTS (select 1 from vm_static where vm_guid = '00000000-0000-0000-0000-000000000000' AND default_display_type = 1);
+
+-- Removing all video devices from blank template
+DELETE FROM vm_device
+WHERE type = 'video'
+    AND vm_id = '00000000-0000-0000-0000-000000000000'
+    AND EXISTS (select 1 from vm_static where vm_guid = '00000000-0000-0000-0000-000000000000' AND default_display_type = 1);
+
+-- Add VGA video device to blank template
+INSERT INTO vm_device (device_id, vm_id, type, device, address, spec_params, is_managed, is_plugged, is_readonly)
+SELECT uuid_generate_v1(), '00000000-0000-0000-0000-000000000000', 'video', 'vga', '', '{"vram" : "16384"}', true, true, false
+WHERE EXISTS (select 1 from vm_static where vm_guid = '00000000-0000-0000-0000-000000000000' AND default_display_type = 1);
+
+-- Change InstanceType's and Blank template default display type from QXL (1) to VGA (2)
```

Comment:
are you sure we then need anything else for REST API? At least for a Blank-based VMs REST API seems to be created with VGA.

Reply:
...and I think it should be changed in 01100_insert_vm_static.sql as well

Reply:
You said it works for API, the last part i don't think it's a must but i did it anyway.
API will be tested nonetheless.